### ISSUE TITLE
[Accessibility], WCAG 1.3.1, 4.1.3. The screen reader does not read the notifications that appear in connection with the field

### ISSUE THREAD

**Issue by Eiskore** [NatLibFi/Skosmos#1993]
## Description of the problem
If a term that is already in the vocabulary is entered to the form field, a notification appears under the field. However, the screen reader does not read the notification that has appeared on the screen.

**Fix**: When error messages / similar notifications that require attention appears, they must also be brought to the attention of screen readers as soon as they appear. Attach the notifications to the input field with the aria-describedby attribute. Add the notification to the live area as well. In this context, it is recommended to use aria-live="polite", in which case the screen reader does not interrupt the reading when the notification appears, but reads it when the reading of the previous content has ended.

**How to test:** Turn on NVDA. Type some term (eg. Testaus) to main term field. When notification appears below the field, don’t move screen reader focus. Just listen how screen reader informs the user about the notification

Expected result: Screen reader should read the notification when it appers, without moving the focus.

Severity: moderate
Priority: medium

## Additional information (e.g. screenshots) about the problem

<img width="1004" height="227" alt="Image" src="https://github.com/user-attachments/assets/dca49795-1a51-49eb-a009-50a5a08e5cee" />

## The browser you used when the problem appeared
Chrome, Firefox, Edge
